tests/integration_tests/scheduler/test_openpbs_driver.py

```diff
@@ -107,3 +108,39 @@ def test_openpbs_driver_with_poly_example():
         "--enable-scheduler",
         "poly.ert",
     )
+
+
+@pytest.mark.timeout(180)
+@pytest.mark.integration_test
+@pytest.mark.usefixtures("copy_poly_case")
+def test_openpbs_driver_with_poly_example_failing_submit_fails_ert_and_propagates_exception_to_user(
+    monkeypatch, caplog
+):
+    monkeypatch.setattr(OpenPBSDriver, "submit", lambda *args, **kwargs: 1 / 0)
```

Comment:
Consider moving this test to `unit_tests`. This test cannot use the actual OpenPBS cluster so this should instead use `MockDriver`.

Also, consider not using `1 / 0` but throwing an error like `raise RuntimeError("Testing")`